context.go

```diff
@@ -103,6 +104,12 @@ func (dc *Context) SavePNG(path string) error {
 	return SavePNG(path, dc.im)
 }
 
+// EncodePNG encodes the image as a PNG and
+// writes it to the provided io.Writer.
+func (dc *Context) EncodePNG(w io.Writer) error {
+	return EncodePNG(w, dc.im)
```

Comment:
Although just to confirm, do we want to encourage people to always use the Context methods? I was wondering if exposing the helper function would serve the same purpose as SavePNG does, ie usable by users of the package.